typescript/hardhat/package.json

```diff
@@ -16,7 +16,7 @@
     "test": "test"
   },
   "dependencies": {
-    "@abacus-network/core": "^0.0.5",
+    "@abacus-network/core": "^0.0.6",
```

Comment:
was this needed?

Reply:
The API changes for [TestAbacusDeploy.ts](https://github.com/abacus-network/abacus-monorepo/pull/341/files#diff-ca6b384d7e6dd7ed8ff62341a817062118ec47b524806459f49d3b6c5bc02ff6). Packages in the monorepo should depend on the latest package version of other dependencies in the repo, and someone missed it here when bumping core before

Reply:
I wonder who that "someone" was 🙃